reactor-core/src/main/java/reactor/core/publisher/ContextPropagationSupport.java

```diff
@@ -0,0 +1,62 @@
+/*
+ * Copyright (c) 2023 VMware Inc. or its affiliates, All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *   https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package reactor.core.publisher;
+
+import reactor.util.Logger;
+import reactor.util.Loggers;
+
+final class ContextPropagationSupport {
+    static final Logger LOGGER = Loggers.getLogger(ContextPropagationSupport.class);
+
+    // Note: If reflection is used for this field, then the name of the field should end with 'Available'.
+    // The preprocessing for native-image support is in Spring Framework, and is a short term solution.
+    // The field should end with 'Available'. See org.springframework.aot.nativex.feature.PreComputeFieldFeature.
+    // Ultimately the long term solution should be provided by Reactor Core.
+    static final boolean isContextPropagationOnClasspath;
```

Comment:
is it intentionally that comment says field should end on `Available` and this one does not?

Reply:
no, it is not intentional. I actually tried to reduce the modifications as much as possible in order to avoid regressions; so I kept this comment that was already present in the current version [here](https://github.com/reactor/reactor-core/blob/main/reactor-core/src/main/java/reactor/core/publisher/ContextPropagation.java#L52-L56).

honestly, I don't know if this comment should be changed , or even removed, or we we need to keep it ?

Reply:
I see